src/main/java/servlets/calculator.java

```diff
@@ -0,0 +1,158 @@
+                if (test(parameterLetter)) {
+                    parameterLetter = req.getParameter(parameterLetter);
+                }
+                equationWtk.append(parameterLetter);
+            } else {
+                equationWtk.append(equation.charAt(i));
+            }
+        }
+        out.print(get(equationWtk.toString()));
+        out.flush();
+        out.close();
+    }
+
+    private static boolean test(String test) {
+        Pattern p = Pattern.compile("[a-z]");
+        Matcher m = p.matcher(test);
+        return m.matches();
+    }
+
+    private int get(String result){
+        String equation = preparation(result);
+        StringBuffer current = new StringBuffer();
+        ArrayList<Integer> Value = new ArrayList<>();
+        int end = 0;
+        for(int i = 0; i < equation.length();i++){
+            if(whatIsOperator(equation.charAt(i))){
+                switch (equation.charAt(i)) {
+                    case '+':
+                        if(Value.size() >= 2){
+                            end = Value.get(1) + Value.get(0);
+                            Value.remove(0);
+                            Value.remove(0);
+                            Value.add(0, end);
+                        }
+                        break;
+                    case '-':
+                        if(Value.size() >= 2){
+                            end = Value.get(1) - Value.get(0);
+                            Value.remove(0);
+                            Value.remove(0);
+                            Value.add(0, end);
+                        }
+                        break;
+                    case '/':
+                        if(Value.size() >= 2){
+                            end = Value.get(1) / Value.get(0);
+                            Value.remove(0);
+                            Value.remove(0);
+                            Value.add(0, end);
+                        }
+                        break;
+                    case '*':
+                        if(Value.size() >= 2){
+                            end = Value.get(1) * Value.get(0);
+                            Value.remove(0);
+                            Value.remove(0);
+                            Value.add(0, end);
+                        }
+                        break;
+                    default:
+                        break;
+                }
+            }else if(equation.charAt(i) != ' '){
+                current.append(equation.charAt(i));
+                if(equation.charAt(i+1) == ' ') {
+                    Value.add(0, Integer.parseInt(String.valueOf(current)));
+                }
+            }else {
+                current.replace(0, current.capacity(), "");
+            }
+        }
+        return end;
+    }
+    
+    private boolean whatIsOperator(char ch) {
+        return ch == '*' || ch == '-' || ch == '/' || ch == '+' ;
+    }
+
+    private int isThat_D_or_M(char ch) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The instance method name 'isThat_D_or_M' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4854440)